### PR TITLE
Homework7: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**
homework_7

### CODE REVIEW

**Comment by ETsiunel** [comment]
@Stml89 PR is ready for review

File: hw_5.py
Comment (Stml89) [inline hw_5.py:1]: this file is related to prev. hw

File: hw_7.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+# чтобы каждая статуя была больше предыдущей ровно на 1.
+# Для этого Вам могут понадобиться дополнительные статуи.
+# Определите количество отсутствующих статуй.
+# Пример
+# Для статуй = [6, 2, 3, 8] результат должен быть = 3.
+# Иными словами, у Вас отсутствуют статуи размеров 4, 5 и 7.
+
+gift = [6, 2, 3, 8]
+new_gift = list(range(min(gift), max(gift)+1))
+# print(new_gift)
```
Comment (Stml89) [inline hw_7.py]: pls do not leave commented line in work repository

File: hw_7.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        return True
+    return False
+
+
+# Генерация случайного 4-х значного числа
+comp_numb_gen = 0  # число загаданное компьютером
+while not computer_number_check(comp_numb_gen):
+    comp_numb_gen = random.randint(1000, 9999)
+print(f"Computer number: {comp_numb_gen}")
+# just debug print
```
Comment (Stml89) [inline hw_7.py]: do not comment obvious actions

File: hw_7.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+pl_num_attempt = player_number_check()
+# 1-й вызов функции подсчета быков и коров
+bulls_guess, cows_guess = bulls_cows_counter(comp_numb_gen, pl_num_attempt)
+print("You guessed", bulls_guess, "bull(s) and", cows_guess, "cow(s).")
+
+# Цикл повтора вышеуказанных функций пока количество быков не станет == 4
+while bulls_cows_counter(comp_numb_gen, pl_num_attempt) != (4, 0):
+    print("    Try one more time!")
+    pl_num_attempt = player_number_check()
+    bulls_guess, cows_guess = bulls_cows_counter(comp_numb_gen, pl_num_attempt)
```
Comment (Stml89) [inline hw_7.py:66]: lines #66-68 the same with lines# 74-75

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hw_5.py
Comment (ETsiunel) [reply hw_5.py:1]: yes, there was 1 small grammatical mistake

File: hw_7.py (hunk 1 above)
Comment (ETsiunel) [reply hw_7.py]: fixed

File: hw_7.py (hunk 2 above)
Comment (ETsiunel) [reply hw_7.py]: fixed

File: hw_7.py (hunk 3 above)
Comment (ETsiunel) [reply hw_7.py:66]: fixed

**Review by Stml89 — APPROVED** [review]

**Pull request merged**